Workbooks/Azure Advisor/Reliability/ResiliencyScenarios/ReliabilityWorkbookResiliencyScenarios.workbook

```diff
@@ -0,0 +1,1779 @@
+                    "text": "{0}{1}"
+                  }
+                ],
+                "customColumnWidthSetting": "30ch"
+              },
+              "numberFormat": {
+                "unit": 1,
+                "options": {
+                  "style": "decimal",
+                  "minimumFractionDigits": 1
+                }
+              }
+            },
+            {
+              "columnMatch": "Total Resources",
+              "formatter": 3,
+              "formatOptions": {
+                "palette": "blue"
+              }
+            },
+            {
+              "columnMatch": "count_",
+              "formatter": 0,
+              "formatOptions": {
+                "aggregation": "Sum"
+              },
+              "numberFormat": {
+                "unit": 0,
+                "options": {
+                  "style": "percent",
+                  "minimumIntegerDigits": 1
+                }
+              }
+            }
+          ],
+          "hierarchySettings": {
+            "treeType": 1,
+            "groupBy": [
+              "Subscription"
+            ],
+            "expandTopLevel": true,
+            "finalBy": "Subscription"
+          },
+          "sortBy": [
+            {
+              "itemKey": "$gen_thresholds_RegionRedundancy_7",
+              "sortOrder": 2
+            }
+          ],
+          "labelSettings": [
+            {
+              "columnId": "RegionRedundancy",
+              "label": "Cross Region Replication"
+            }
+          ]
+        },
+        "sortBy": [
+          {
+            "itemKey": "$gen_thresholds_RegionRedundancy_7",
+            "sortOrder": 2
+          }
+        ],
+        "tileSettings": {
+          "showBorder": false,
+          "titleContent": {
+            "columnMatch": "Service",
+            "formatter": 1
+          },
+          "leftContent": {
+            "columnMatch": "count_",
+            "formatter": 12,
+            "formatOptions": {
+              "palette": "auto"
+            },
+            "numberFormat": {
+              "unit": 17,
+              "options": {
+                "maximumSignificantDigits": 3,
+                "maximumFractionDigits": 2
+              }
+            }
+          }
+        },
+        "graphSettings": {
+          "type": 0,
+          "topContent": {
+            "columnMatch": "Service",
+            "formatter": 1
+          },
+          "centerContent": {
+            "columnMatch": "count_",
+            "formatter": 1,
+            "numberFormat": {
+              "unit": 17,
+              "options": {
+                "maximumSignificantDigits": 3,
+                "maximumFractionDigits": 2
+              }
+            }
+          }
+        }
+      },
+      "customWidth": "100",
+      "name": "Details - Display"
+    }
+  ],
+  "fallbackResourceIds": [
+    "Azure Monitor"
+  ],
+  "fromTemplateId": "community-Workbooks/Azure Advisor/AzureServiceRetirement",
```

Comment:
fromTemplateId this field shouldnt be present
the validation failed due to that

Reply:
I fixed this part. Could you check again? > @pkrishnanms